sdk/test-utils/recorder/src/utils/index.ts

```diff
@@ -436,8 +436,39 @@ export function handleSingleQuotesInUrlPath(fixture: string): string {
   return updatedFixture;
 }
 
+/**
+ * Meant for node recordings only!
+ *
+ * Masks access tokens in the json response from nock fixtures.
+ * For example, the following part of the nock fixture/recording would be updated.
+ * from `.reply(200, {"token_type":"Bearer","expires_in":86399,"access_token":"e6z-9_g"}, [`
+ * to   `.reply(200, {"token_type":"Bearer","expires_in":86399,"access_token":"access_token"}, [`
+ *
+ * @param {string} fixture
+ */
+export function maskAccessTokenInNockFixture(fixture: string): string {
+  // Replaces only if the content-type is json
+  if (!isBrowser() && isContentTypeInNockFixture(fixture, jsonContentTypes)) {
+    // Matches the nock's reply from the fixture
```

Comment:
It's nice to put a sample of what you're matching above the regex:

```typescript
// ex: .reply(200, {"token_type":"Bearer","expires_in":86399,"ext_expires_in":86399,"access_token":"sometoken"}, [
```

Or something like that.

Reply:
Added it in the JS docs, can add here too.